proto/dydxprotocol/indexer/events/events.proto

```diff
@@ -423,6 +423,10 @@ message PerpetualMarketCreateEventV2 {
 
   // Market type of the perpetual.
   dydxprotocol.indexer.protocol.v1.PerpetualMarketType market_type = 11;
+
+  // Since: v8.x
+  // Default 8hr funding rate in parts-per-million.
+  int32 default_funding8hr_ppm = 12;
```

Comment:
Note: Instead of Harry's approach which creates a new V2 message type, simply add the field which is non-breaking. The event handler just needs handle the case when the field is absent.